src/main/java/de/muenchen/isi/infrastructure/entity/stammdaten/SobonJahr.java

```diff
@@ -0,0 +1,30 @@
+package de.muenchen.isi.infrastructure.entity.stammdaten;
+
+import de.muenchen.isi.infrastructure.entity.BaseEntity;
+import java.time.LocalDate;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Entity
+@Data
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class SobonJahr extends BaseEntity {
```

Comment:
Weshalb wird die Entität `SobonJahr` benannt und enthält ein Attribut `jahr`. 

Die für die Berechnung zu wählenden Stammdaten sind mittels des Attributs `gueltigAb` ermittelbar.

Beispiel:
- Stammdatum A → Gültig ab 01.01.2020
- Stammdatum B → Gültig ab 01.08.2025


Das Stammdatum A ist somit vom 01.01.2020 bis einschließlich 31.07.2025 gültig. Wird bei der Berechnung anstelle des "Jahr für Sobon-Orientierungswerte" eine Datumsangabe 01.01.2022 (Abgeleitet aus dem in der GUI gewählten Sobon-Jahr)  gewählt, so wird für die Berechnung das Stammdatum A verwendet.  Wird als Datumsangabe 06.06.2030 gewählt, so wird für die Berechnung das Stammdatum B angezogen.